src/components/forms/SearchBar/SearchBar.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { act, fireEvent, render, screen } from '@testing-library/react';
+
+import SearchBar from './SearchBar';
+
+const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+
+const onClickSuggestion = jest.fn();
+
+export const mockOnSearch = async (query) => {
+  await delay(20);
+  return [
+    {
+      name: `${query} suggestion 1`,
+      value: `${query} suggestion 1`,
+      filter: { field: 'Domain', condition: 'is' },
+      onClick: onClickSuggestion,
+    },
+    {
+      name: `${query} suggestion 2`,
+      value: `${query} suggestion 2`,
+      filter: { field: 'Domain', condition: 'is' },
+      onClick: onClickSuggestion,
+    },
+    {
+      name: `${query} suggestion 3`,
+      value: `${query} suggestion 3`,
+      filter: { field: 'Domain', condition: 'contains' },
+      onClick: onClickSuggestion,
+    },
+  ];
+};
+
+const onSearch = jest.fn(mockOnSearch);
+const setup = () => {
+  const utils = render(
+    <SearchBar
+      defaultValue="Searching for Default"
+      placeholder="Search for X"
+      onSearch={onSearch}
+    />,
+  );
+  const searchInput = utils.getByPlaceholderText('Search for X');
+  return {
+    searchInput,
+    ...utils,
```

Comment:
This isn't used anywhere, why exporting it?

Reply:
fixed in last commit

Reply:
I still see `...utils` being exported but not used 🤔